tests/api.node-spec.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { render } from 'enzyme';
+import { JSDOM } from 'jsdom';
+
+const dom = new JSDOM('');
+const customWindow = dom.window;
+
+import * as htmlStringToReact from '../src/index';
+
+describe('Public NodeJS API', () => {
+  it('should convert a string to an array of react nodes with custom window', () => {
```

Comment:
Can we add a test with no custom window to test if it is getting the default `window` correctly?